### DISCUSSION TITLE
anderson tigard

### DISCUSSION THREAD

**Discussion by defundinformants** [community/community#65641] [Discussions] [answered]
there is a project on here that is targeting me and the person in charge named taylor nicole anderson is posing as me, emily k. taylor is a fraud, scam artist and a liar.she is using this platform to harass someone who is innocent and that person is going to press charges on EVERYONE if they do not stop harassing her, video taping her without her consent, breaking into her apartment, spying on her, turning people against her, harassing her and ruining her days. this is not a joke. leave emily alone

**Comment by Akash1134** [comment, marked as answer]
Hi @bbb11223344556677 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
